modules/local/m6anet_inference.nf

```diff
@@ -0,0 +1,26 @@
+process M6ANET_INFERENCE {
+    tag "$meta.id"
+    echo true
+    label 'process_medium'
+
+//  conda     (params.enable_conda ? "bioconda::nanopolish==0.13.2" : null) // need to get xpore onto conda
```

Comment:
No conda or singularity?

Reply:
Our lab is trying to get `m6anet` onto `bioconda`, but the person who wrote `m6anet` is very busy right now, so I guess we will include `conda` and `singularity` in a future release instead.